src/scss/index.scss

```diff
@@ -1 +1,2 @@
-
+@import "variables";
```

Comment:
Can you show an implementation of the variable you override? You need to use it in your design else, why do you override it in the first place